app/controllers/api/v1/webhooks_controller.rb

```diff
@@ -25,4 +45,12 @@ def twitter_client
   def twitter_consumer
     @twitter_consumer ||= ::Webhooks::Twitter.new(params)
   end
+
+  def instagram_consumer
```

Comment:
let decouple the logic from this controller into a dedicated controller for Instagram